src/coreclr/src/tools/ReadyToRun.SuperIlc/CpaotRunner.cs

```diff
@@ -116,11 +119,15 @@ private List<string> ResolveReferences()
         {
             char referenceOption = (_options.Composite ? 'u' : 'r');
             List<string> references = new List<string>();
-            foreach (var referenceFolder in _referenceFolders)
+            foreach (string referenceFolder in _referenceFolders)
```

Comment:
A nit - It looks like it would be nice to unify this with the same code starting at line 97.

Reply:
Good point, thanks. Will do!